examples/msal-net/akvdotnet/TokenCredential.cs

```diff
@@ -55,6 +58,7 @@ public override async ValueTask<AccessToken> GetTokenAsync(TokenRequestContext r
         return new AccessToken(result.AccessToken, result.ExpiresOn);
     }
 
+    // This assumes the file is always updated with a fresh JWT 
```

Comment:
The token lifetime is between 1-24h, so it's not always updated.

Reply:
Ok, as long as there is some buffer time it should be fine. In MSAL we take 5 min buffer.